docs/json-examples.md

```diff
@@ -0,0 +1,744 @@
+### All
+
+JSON sent with every notification:
+
+```json5
+{
+  "content": "Text message as set by the user",
+  "extra": {},
+  "type": "NOTIFICATION_TYPE",
+  "playerName": "your rsn",
+  "accountType": "NORMAL | IRONMAN | HARDCORE_IRONMAN",
+  "dinkAccountHash": "abcdefghijklmnopqrstuvwxyz1234abcdefghijklmnopqrstuvwxyz",
+  "embeds": []
+}
+```
+
+JSON sent with every notification but only in certain circumstances:
+
+```json5
+{
+  "clanName": "Dink QA",
+  "groupIronClanName":"Dink QA",
+  "discordUser":{
+    "id":"012345678910111213",
+    "name":"Gamer",
+    "avatarHash":"abc123def345abc123def345abc123de"
+   },
+```
+
+`clanName` is only sent when the player is in a clan and has the advanced setting `Send Clan Name` enabled.  
+`groupIronClanName` is only sent when the player is a GIM and has the advanced setting `Send GIM Clan Name` enabled.  
+The `discordUser` object is only sent when Discord is open and the advanced setting `Send Discord Profile` is enabled.
+
+Note: The examples below omit `playerName`, `accountType`, and `dinkAccountHash` keys because they are always the same. Also, `embeds` is excluded for simplicity.
```

Comment:
```suggestion
Note: The examples below omit `playerName`, `accountType`, and `dinkAccountHash` keys because they are always the same.

Also, `content` is populated instead of `embeds` for simplicity; this would correspond to the advanced setting `Use Rich Embeds` being disabled. Third-party integrations should rely on the `extra` object instead of `content`/`embeds`.
```